shrine/lib/supplemental/product_columns.dart

```diff
@@ -27,8 +27,7 @@ class TwoProductCardColumn extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return LayoutBuilder(
-        builder: (BuildContext context, BoxConstraints constraints) {
+    return LayoutBuilder(builder: (context, constraints) {
```

Comment:
optional nit: I would throw a comma between the brace and parenthesis on line 63 to bump the builder parameter down a line and maintain the one-closing-bracket-per-line indent format.

Reply:
Amusingly, that minor change caused a full relayout of pretty much everything.